scalability_and_performance/optimization/optimizing-networking.adoc

```diff
@@ -6,8 +6,6 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The xref:../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShift SDN] uses OpenvSwitch, virtual extensible LAN (VXLAN) tunnels, OpenFlow rules, and iptables. This network can be tuned by using jumbo frames, multi-queue, and ethtool settings.
-
 xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes] uses Generic Network Virtualization Encapsulation (Geneve) instead of VXLAN as the tunnel protocol. This network can be tuned by using network interface controller (NIC) offloads.
```

Comment:
The "instead of" here is a dangling reference after you remove the sdn section. Though "Geneve" is still kind of new and admins might not know about it, so maybe "OVN-Kubernetes uses Geneve ("Generic Network Virtualization Encapsulation", a protocol similar to VXLAN) to tunnel traffic between nodes."

Also, the rest of the doc only talks about VXLAN. You should change all of the VXLAN references to say "Geneve" instead.